wallet/wallet.go

```diff
@@ -44,9 +44,9 @@ func (m *Module) NewAddress(ctx context.Context, typ string) (string, error) {
 		return "", err
 	}
 
-	if m.masterAddr != nil {
+	if m.masterAddr != address.Undef {
```

Comment:
Here was a little problem. `address.NewFromString` returns an `Undef` address, not an error nor nil.